sphinx/ext/autodoc/__init__.py

```diff
@@ -752,9 +752,11 @@ def document_members(self, all_members=False):
         members_check_module, members = self.get_object_members(want_all)
 
         # remove members given by exclude-members
-        if self.options.exclude_members:
+        exclude = self.options.exclude_members or \
```

Comment:
nit: I'd rather see this wrapped in brackets as it reads a little better. However, we use `/` elsewhere in the code base so it's not a pressing issue